src/sagemaker/tensorflow/README.rst

```diff
@@ -743,26 +293,11 @@ You can disable MKL-DNN optimization for TensorFlow ``1.8.0`` and above by setti
 SageMaker TensorFlow Docker containers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The TensorFlow Docker images support Python 2.7. They include the following Python packages:
-
-- boto3
-- botocore
-- CUDA 9.0 (GPU image only)
-- grpcio
-- numpy
-- pandas
-- protobuf
-- scikit-learn
-- scipy
-- sklearn
-- tensorflow
-- tensorflow-serving-api
```

Comment:
it could be nice to turn this into the table that we have with some of the other frameworks.